src/Requests.jl

```diff
@@ -14,6 +14,7 @@ import URIParser: URI
 import HttpCommon: Cookie
 
 using Compat
+using Dates
```

Comment:
Why do we need this?

Reply:
Changed it to inline `Compat.Dates` for more clarity. Compat.Dates` is provided on versions older than 0.7, where this library is not yet a part of the standard library.